commit c4088252ee68
Author: Henri Sivonen <hsivonen@hsivonen.fi>
Date:   Wed Feb 28 13:53:29 2018 +0200

    Bug 1425520. r=smaug. a=abillings
    
    MozReview-Commit-ID: IoZeR5BGjmQ
    
    --HG--
    extra : source : e20052aa341d6da4505eb84be6e061bb7f37de51
---
 dom/base/nsDocumentEncoder.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git dom/base/nsDocumentEncoder.cpp dom/base/nsDocumentEncoder.cpp
index d591f61c6244..9c0eaaa63bc4 100644
--- dom/base/nsDocumentEncoder.cpp
+++ dom/base/nsDocumentEncoder.cpp
@@ -517,7 +517,7 @@ nsDocumentEncoder::SerializeToStringRecursive(nsINode* aNode,
   }
 
   if (!aDontSerializeRoot) {
-    rv = SerializeNodeEnd(node, aStr);
+    rv = SerializeNodeEnd(maybeFixedNode, aStr);
     NS_ENSURE_SUCCESS(rv, rv);
   }
 
